cinema/urls.py

```diff
@@ -1 +1,23 @@
-# write urls here
+from django.urls import include, path
+from rest_framework.routers import DefaultRouter
+
+from cinema.views import (
+    GenreViewSet,
+    ActorViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet
+)
+
+router = DefaultRouter()
+router.register(r"genres", GenreViewSet)
+router.register(r"actors", ActorViewSet)
+router.register(r"cinema_halls", CinemaHallViewSet)
+router.register(r"movies", MovieViewSet)
+router.register(r"movie_sessions", MovieSessionViewSet)
+
+urlpatterns = [
+    path("", include(router.urls)),
+]
```

Comment:
No need to include if basepath is not provided. Instead use:
urlpatters = router.urls

Reply:
@xtina999 not fixed